int32 vicon_marker_number